src/tini.c

```diff
@@ -164,6 +165,13 @@ void print_license(FILE* const file) {
 int parse_args(const int argc, char* const argv[], char* (**child_args_ptr_ptr)[], int* const parse_fail_exitcode_ptr) {
 	char* name = argv[0];
 
+	// We handle --version if it's the *only* argument provided.
+	if (argc == 2 && strcmp("--version", argv[1]) == 0) {
+		*parse_fail_exitcode_ptr = 0;
+		fprintf(stdout, "%s\n", TINI_VERSION_STRING);
+		return 1;
```

Comment:
@mlaventure it won't (the tests in `run_build.sh` check against that since `set -o pipefail` is enabled). 

This piece of code can be a little confusing, but the idea is that the return value of `parse_args` indicates whether `main` should continue after parsing arguments: if it's non-zero, then `main` will exit with whatever `parse_fail_exitcode` was set to, and if it's zero then `main` will continue.

`parse_fail_exitcode` is 1 by default, but we set it to 0 here via `*parse_fail_exitcode_ptr = 0;`, so Tini will exit with 0 😄